compiler/front_end/module_ir_test.py

```diff
@@ -4025,29 +3956,25 @@ def _check_source_location(source_location, path, min_start, max_end):
     result = []
     start = None
     end = None
-    if not source_location.HasField("start"):
+    if not source_location.start:
         result.append("{}.start missing".format(path))
     else:
         start = source_location.start
-    if not source_location.HasField("end"):
+    if not source_location.end:
         result.append("{}.end missing".format(path))
     else:
         end = source_location.end
 
     if start and end:
-        if start.HasField("line") and end.HasField("line"):
+        if start.line and end.line:
             if start.line > end.line:
                 result.append(
                     "{}.start.line > {}.end.line ({} vs {})".format(
                         path, path, start.line, end.line
                     )
                 )
             elif start.line == end.line:
-                if (
-                    start.HasField("column")
-                    and end.HasField("column")
-                    and start.column > end.column
-                ):
+                if start.column and end.column and start.column > end.column:
```

Comment:
nit: we shouldn't have the `start.column and end.column` portion here, they'll always be present, this is just checking that they're not zero now right?